setup.py

```diff
@@ -37,6 +37,12 @@
     ('/opt/intelmq/var/lib/bots/file-output/',
      [],
      ),
+    ('/usr/bin',
```

Comment:
It is bad practise to install something to `/usr/local/` from a package. This Folder is intended for manual "installations" w/o packages.